utils/utils.go

```diff
@@ -223,3 +228,22 @@ func UploadFileToS3(file *multipart.FileHeader) (*manager.UploadOutput, error) {
 
 	return result, nil
 }
+
+// Helper function to determine content type based on file extension
+func getContentType(filename string) string {
+	ext := strings.ToLower(filepath.Ext(filename))
+	switch ext {
+	case ".jpg", ".jpeg":
+		return "image/jpeg"
+	case ".png":
+		return "image/png"
+	case ".gif":
+		return "image/gif"
+	case ".webp":
+		return "image/webp"
+	case ".ply":
+		return "application/vnd.ply"
+	default:
+		return "application/octet-stream"
```

Comment:
I actually a little bit concern with this part. I think some could upload files with extension .exe, .bat or some malicious file type too.